src/main/java/com/example/controller/GraphController.java

```diff
@@ -196,6 +218,25 @@ public void initGraphView() {
         }
     }
 
+    public void exportGraph() throws IOException {
```

Comment:
Methods import and export could easily be extracted from GraphController. These methods only need to get and set graph from/to GraphController. Graph Controller is already a huge class and definitely adding next responsibility is not a good idea. All of this logic could be easily moved to FileMenuController, which I mentioned before.